### PR TITLE
Fix doc

### PR DESCRIPTION
**Pull request by minggo**

### CODE REVIEW

File: cocos/primitive/cylinder.ts [hunk 1]
```diff
@@ -50,9 +48,9 @@ const temp2 = new Vec3(0, 0, 0);
...
  * @zh
  * 生成一个圆柱。
- * @param radiusTop 顶部半径。
- * @param radiusBottom 底部半径。
- * @param opts 圆柱参数选项。
+ * @param radiusTop @en Radius of top plane. @zh 顶部半径。
+ * @param radiusBottom @en Radius of bottom plane.  @zh 底部半径。
+ * @param opts @en Options of cylinder.  @zh 圆柱参数选项。
  */
 export default function cylinder (radiusTop = 0.5, radiusBottom = 0.5, height = 2, opts: RecursivePartial<ICylinderOptions> = {}): IGeometry {
```
Comment (stanleyljl) [inline cocos/primitive/cylinder.ts:56]: Do you leak height param?

Comment (minggo) [reply cocos/primitive/cylinder.ts:56]: Yep, i only modified exist comments. I will add it. 
Good catch.

Comment (minggo) [reply cocos/primitive/cylinder.ts:56]: I fixed it.

**Review by stanleyljl — APPROVED** [review]

**Comment by pandamicro** [comment]
https://github.com/cocos/cocos-engine/issues/10840

**Pull request merged**